source/extensions/common/wasm/wasm_vm.cc

```diff
@@ -72,11 +72,27 @@ bool EnvoyWasmVmIntegration::getNullVmFunction(std::string_view function_name, b
   return false;
 }
 
+absl::string_view getWasmEngine() {
```

Comment:
Nit: `getDefaultWasmEngineName` or similar might be more appropriate, since we're not getting an instance here.